src/rt/sched/backpressure.h

```diff
@@ -0,0 +1,249 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#pragma once
+
+/**
+ * ## Introduction
```

Comment:
@rengolin Renato, do you think this would have been easier for you if all the terminology like "sender", "receiver", "muted", "overloaded", "mutor" etc had been defined up front before the introduction?

Reply:
Those terms specifically were pretty clear to me what they meant from the start, but perhaps I'm already biased by being at the meetings where theses things are being discussed. :)

Thogh, my personal preference would be to have a document explaining all of those things and not in a header comment. We have a docs directory for that, would be nice to populate it with some better documents aimed at developers and users.

Reply:
That makes sense @rengolin. I think then we should either include that document as part of this PR or open an issue for the creation of that document and our general expectations of what should be in it.

Reply:
Perhaps most of the discussion here and most of this comment could go inside such document, and here a quick explanation and a link to the document.

Reply:
That sounds quite reasonable to me.